docs/modules/plugins/pages/plugin-lighthouse.adoc

```diff
@@ -0,0 +1,61 @@
+= Lighthouse Plugin
+
+The plugin provides the ability to generate Lighthouse performance reports on the user experience of a page on both mobile and desktop devices, and provides suggestions on how that page may be improved.
+
+:plugin-short-name: lighthouse
+include::partial$plugin-installation.adoc[]
+
+== Properties
+[cols="2,1,2,3", options="header"]
+|===
+|Property Name
+|Acceptable values
+|Default
+|Description
+
+|`lighthouse.application-name`
+|<string>
+|
+|The application name to be used in the https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent[User-Agent] header.
+
+|`lighthouse.access-token`
+|<string>
+|
+|The OAuth access token. Its recommended to specify the token if you plan to use the plugin in an automated way and making multiple queries per second.
+
+|===
+
+== Steps
+
+=== Perform Lighthouse scan
+
+Performs Lighthouse scan on the specified page and verifies performance result metrics.
+
+[source,gherkin]
+----
+When I perform Lighthouse scan on $strategy for `$url` page:$rules
+----
+
+* `$strategy` - The analysis strategy to use, either `desktop` or `mobile`.
+* `$url` - The page URL to perform scan on.
+* `$rules` - The metric verification rules:
+** `$metric` - The Lighthouse metrics such as:
+*** `First Contentful Paint`
+*** `Largest Contentful Paint`
+*** `Total Blocking Time`
+*** `Cumulative Layout Shift`
+*** `Speed Index`
+** `$rule` - xref:parameters:comparison-rule.adoc[The comparison rule].
+** `$value` - The expected numeric metric value.
```

Comment:
not clear what `numeric` is? is it integer/floating?

Reply:
numeric value refers to both integers and floating numbers